source/features/hide-empty-split-diff-side.tsx

```diff
@@ -0,0 +1,23 @@
+import './hide-empty-split-diff-side.css';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	for (const diffTable of select.all('.js-diff-table')) {
+		for (const side of ['left', 'right']) {
+			if (!select.exists(`[data-split-side="${side}"]:is(.blob-code-addition, .blob-code-deletion)`, diffTable)) {
+				diffTable.classList.add('rgh-empty-split-' + side);
+				break;
+			}
+		}
+	}
+}
+
+void features.add(__filebasename, {
+	include: [
+		() => pageDetect.isPRFiles() && select.exists('[value="split"][checked]')
```

Comment:
> Maybe this could be applied to everywhere split diffs exists, not just PRs: commit pages, compare pages, new PR creation, etc.?

Yes of course

> maybe `pageDetect` could have `isSplitDiffView()`

That's a little specific, it's not something that generally belongs there.

Also:
```suggestion
		pageDetect.isPRFiles
	],
	exclude: [
		() => !select.exists('[value="split"][checked]')
```

Related: https://github.com/sindresorhus/refined-github/issues/4008#issuecomment-812843149